automation/submission-perf-test.sh

```diff
@@ -172,11 +173,11 @@ if [ $ENV = "alpha" ]; then
     monitorSubmission dumbledore.admin@test.firecloud.org aa-test-042717a test-042717 $test1
 
    i=1
-   while [ "$submissionStatus" != "Done" ] && [ "$i" -le 19 ]
+   while [ "$submissionStatus" != "Done" ] && [ "$i" -le 199 ]
```

Comment:
had to increase the number of polls allowed, because I reduced the polling interval on line 180